.github/workflows/ci.yml

```diff
@@ -232,3 +232,81 @@ jobs:
           body: ""
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  bundle-deb:
+    name: Create a *.deb Linux bundle
+    runs-on: ubuntu-latest
```

Comment:
If we build it on latest ubuntu, wouldn't this prevent anyone on an older linux to run it? I.e. it will expect a certain high version of GLIBC to be available, even though we unlikely actually need it. Perhaps, we should use something like `ubuntu-18` here?

Reply:
I think we need much more in the end, here's some good example: https://github.com/rust-lang/rust-analyzer/blob/767d5d3eab237a1fbc10d62b120020b12f6026c5/.github/workflows/release.yaml#L26-L51

We can already add 

```
          - os: ubuntu-20.04
            target: x86_64-unknown-linux-gnu
            code-target: linux-x64
            container: ubuntu:18.04
          - os: ubuntu-20.04
            target: aarch64-unknown-linux-gnu
            code-target: linux-arm64
          - os: ubuntu-20.04
            target: arm-unknown-linux-gnueabihf
            code-target: linux-armhf
```

and whatever ubuntu-18 libc, but this does not seem hard for the community to do, if they have a need for the particular one (maybe, it also needs libc added into the name?).

What's harder is the caching story, so I would rather concentrate on that myself (probably, in standalone PRs).

Reply:
But reading this again, we need to set the base version low enough to get builds more universal?
Sure, let's try that.

Reply:
yep, just this

Reply:
Apparently, it's gone from the runners: https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners

<img width="839" alt="image" src="https://github.com/zed-industries/zed/assets/2690773/a657fd0c-483b-4c98-a8ac-7be4e7e35f98">

The job does not start: https://github.com/zed-industries/zed/actions/runs/8167955494/job/22329289230?pr=8913

But I can use the 20.04 then.